ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -269,7 +269,7 @@ Update = Update
 Update\ Facility = Edit sender
 Upload\ Forms = Upload forms
 Upload\ failed = Upload failed
-Upload\ forms\ help = The file should contain a JSON encoded list of form definitions. Any existing forms will be overwritten.
+Upload\ forms\ help = The file should contain a JSON encoded list of form definitions. You may only upload one app form file at a time and any existing forms will be overwritten.
```

Comment:
This translation doesn't appear to be used anywhere - I think at some point it was mistakenly duplicated to `upload.json.forms.help`. Please pick one to update and use, and delete the other.